quinn/src/endpoint.rs

```diff
@@ -408,6 +408,7 @@ impl State {
                     .write(IoSliceMut::<'a>::new(buf));
             });
         let mut iovs = unsafe { iovs.assume_init() };
+        let mut buffer = BytesMut::with_capacity(1500);
```

Comment:
I think this (and other instances of this) should presumably use some call into quinn-proto to get the applicable MTU instead of hard-coding this.

Reply:
Please check out my latest commit